fluent-syntax/src/ast.js

```diff
@@ -115,10 +115,10 @@ export class MessageReference extends Expression {
   }
 }
 
-export class ExternalArgument extends Expression {
+export class VariableExpression extends Expression {
```

Comment:
This should be called `VariableReference`.

Reply:
Ahh, great catch! I was following https://github.com/projectfluent/fluent.js/issues/206 which has it as `VariableExpression`. Fixing :)

Reply:
My bad. I fixed the checklist in #206.